source/user-manual/reference/ossec-conf/auth.rst

```diff
@@ -36,7 +35,7 @@ Options
 - `limit_maxagents`_
 
 disabled
-^^^^^^^^
+--------
```

Comment:
There is a new approach, standard, or something like that that made you change this? Just asking.

Reply:
I thought it was easier to understand since the force option now is a bigger block, but there's no other reason.